readthedocs/search/tests/test_xss.py

```diff
@@ -8,7 +8,7 @@
 class TestXSS:
 
     def test_facted_page_xss(self, client, project):
-        query = 'XSS'
+        query = '"XSS"'
```

Comment:
Why did this change?

Reply:
It was matching more results

https://github.com/readthedocs/readthedocs.org/blob/7fd916547c0ac8dd70eaddd4a07a2be0846ab062/readthedocs/search/tests/test_xss.py#L19-L19

We do check the content from the result below, so it's safe
https://github.com/readthedocs/readthedocs.org/blob/fb5b8c8a8a29bdaf9123806f6327ccc101419c3c/readthedocs/search/tests/test_xss.py#L32-L32